controllers/transfers.controller.js

```diff
@@ -32,11 +32,43 @@ const debit = async (account_number, amount) => {
   );
 };
 
+/**
+ * Debits an account by an amount
+ *
+ * @param {String} account_number the account number of the account to be debited
+ * @param {Number} amount the amount to be debited
+ */
+const accountCheck = async (account_number, amount) => {
+  const check = await Accounts.findOne({ account_number });
+  if (check.account_balance < amount) {
+    throw new BadRequestError("Insufficient Funds");
+  } else {
+    debit(account_number, amount);
+  }
+};
+
 const transfer = async (req, res) => {
   // debit the sender
   // credit the recipient
   // create the transaction
   // return a response ot throw and error
```

Comment:
Delete these lines, they are not really useful you can move them to the docs string if you need them